clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3742,7 +3743,8 @@ private List<MemberIdentity> getMembersFromGroup(String groupId, String reason)
             MemberIdentity memberIdentity = new MemberIdentity().setReason(reason);
 
             if (member.groupInstanceId().isPresent()) {
-                memberIdentity.setGroupInstanceId(member.groupInstanceId().get());
+                memberIdentity.setGroupInstanceId(member.groupInstanceId().get())
+                        .setMemberId(JoinGroupRequest.UNKNOWN_MEMBER_ID);
```

Comment:
Is it really better? I am not convinced because `UNKNOWN_MEMBER_ID` is the default value anyway. If we follow this train of thoughts, why don't we add `setGroupInstanceId(null)` to the else branch as well?

Reply:
@dajac thanks for this feedback. You are right :)